build.gradle.kts

```diff
@@ -66,6 +71,11 @@ kotlin {
     ios()
 
     sourceSets {
+        all {
+            languageSettings {
+                optIn("kotlin.experimental.ExperimentalNativeApi")
```

Comment:
This is a new opt in requirement added in 1.9.20. We only use a function it applies to in one place in the `nativeTest` source set, but since the compile language version is 1.7 and this annotation didn't exist in 1.7, we have to opt in as a compiler argument.